app/controllers/me/index.js

```diff
@@ -22,7 +22,7 @@ export default Controller.extend({
   emailNotificationsSuccess: false,
 
   setAllEmailNotifications(value) {
-    this.get('ownedCrates').forEach(c => {
+    this.ownedCrates.forEach(c => {
```

Comment:
`ownedCrates` comes from `ownedCrates: alias('model.ownedCrates')`, no get necessary in this file?

Reply:
removed for now